api/resources/graph.py

```diff
@@ -12,6 +12,8 @@
 class Graph(Resource):
     def get(self):
         code = generate_code(request.args["config"])
+        print("This is none")
+        print(code)
```

Comment:
is this just testing? do we want this in main?

Reply:
thanks for catching this lemme fix that